LibreNMS/OS/Traits/UcdResources.php

```diff
@@ -49,4 +50,68 @@ public function discoverProcessors()
             ),
         ];
     }
+
+    public function discoverMempools()
+    {
+        $mempools = collect();
+        $data = snmp_get_multi($this->getDeviceArray(), [
+            'memTotalSwap.0',
+            'memAvailSwap.0',
+            'memTotalReal.0',
+            'memAvailReal.0',
+            'memBuffer.0',
+            'memCached.0',
+        ], '-OQUs', 'UCD-SNMP-MIB');
```

Comment:
it seems `memSysAvail` isn't in the get list, will `oidValid` check at L66 need this ?